.github/workflows/create-github-release.yaml

```diff
@@ -53,17 +53,20 @@ jobs:
           echo -e "${{ steps.changelog.outputs.release_notes }}" > $RELEASE_NOTES_FILE
           echo "release_notes_file=$RELEASE_NOTES_FILE" >> "$GITHUB_OUTPUT"
 
+      - name: Setup pnpm
+        uses: pnpm/action-setup@v4
+
       - name: Setup Node.js
         uses: actions/setup-node@v4
         with:
-          cache: npm
+          cache: pnpm
           node-version: ${{ env.NODE_VERSION }}
 
       - name: Install dependencies
-        run: npm ci
+        run: pnpm install --frozen-lockfile
```

Comment:
I discovered that `--frozen-lockfile` defaults to true in a CI environment.

https://pnpm.io/cli/install#--frozen-lockfile